scripts/gogenerate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+unset -v progdir
+case "${0}" in
+*/*) progdir="${0%/*}";;
+*) progdir=.;;
+esac
+git grep -l '^//go:generate ' | sort -t/ | (
```

Comment:
pipe through the complicated function makes the code hard to read and debug.  recommend using some temporary file to store the output of "git grep  | sort",

Reply:
I agree the three-stage pipe was probably too much.  Split that into two stages in the new utility script.